localization/i18n/pl/OrcaSlicer_pl.po

```diff
@@ -2576,8 +2576,8 @@ msgid ""
 "Choose an AMS slot then press \"Load\" or \"Unload\" button to automatically "
 "load or unload filaments."
 msgstr ""
-"Wybierz gniazdo AMS, a następnie naciśnij przycisk „Ładuj” lub "
-"„Rozładuj” ,aby automatycznie załadować lub wyładować filamenty."
+"Wybierz gniazdo AMS, a następnie naciśnij przycisk „Ładuj” lub „Rozładuj” ,"
+"aby automatycznie załadować lub wyładować filamenty."
```

Comment:
As you have added backslash to quotation marks in many places, shouldn't it be fixed here too? The comma should also be attached to the word "Rozładuj" not "aby".